image/roxctl/rhtap.Dockerfile

```
@@ -0,0 +1,53 @@
+# TODO: can we follow tags?
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.20 as builder
+
+WORKDIR /go/src/github.com/stackrox/rox/app
+
+# TODO: do we want to avoid this by COPYing each required directory separately?
+COPY . .
+
+RUN git config --global --add safe.directory /go/src/github.com/stackrox/rox/app && \
+    # We could add tag fetching to: https://github.com/redhat-appstudio/build-definitions/blob/main/task/git-clone/0.1/git-clone.yaml
+    # TODO: if git fetch fails, check if privileges/file owners are OK, usually only on local build because repo might have moved further.
+    # TODO: this prevents hermetic builds
+    git fetch --tags --force && \
+    mkdir -p image/bin
+
+ENV CI=1 GOFLAGS="" GOTAGS="release"
+
+RUN RACE=0 CGO_ENABLED=1 GOOS=linux GOARCH=$(go env GOARCH) BUILD_TAG=$(make tag) scripts/go-build.sh ./roxctl && \
+    cp bin/linux_$(go env GOARCH)/roxctl image/bin/roxctl
+
+# TODO: can we follow tags?
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest as app
```

[minor and cosmetic] I think, there's no need to give the final stage a name. Up to you.
```suggestion
FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```

---

I prefer it, but looking at other Dockerfiles in stackrox repository, the consensus is to not name the final stage.